sos/report/plugins/ds.py

```diff
@@ -48,6 +48,10 @@ def setup(self):
                 if d[0:5] == 'slapd':
                     certpath = os.path.join("/etc/dirsrv", d)
                     self.add_cmd_output("certutil -L -d %s" % certpath)
+                    if "ds8" in self.check_version():
```

Comment:
`ds8` is returned by `check_version()` if package `redhat-ds-base` is installed. It's RHDS 8 (RHEL 5), where the healthcheck is not available. I think we should extend `check_version()` to support recent versions of RHDS to correctly use the healthcheck tool.